### DISCUSSION TITLE
Class "chillerlan\QRCode\Common\EccLevel" not found

### DISCUSSION THREAD

**Discussion by tidcp** [chillerlan/php-qrcode#100] [Q&A]
I installed this as part of a Laravel project, and used the svg example file provided. However, I am unable to run the code as cannot find EccLevel

**Environment (please complete the following information):**
 - PHP version/OS: 8.1 (Laravel), macOS Monterey

**Comment by codemasher** [comment]
Looks like you're using the example from the `dev-main` branch in combination with one of the stable releases.
